### PR TITLE
Run and Test Documentation

### LINKED ISSUE #34: No Documentation Exists Specifying How to Run & Test Service

### PR DESCRIPTION
**Pull request by epociask**
## Fixes Issue

Fixes #34

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -18,10 +18,34 @@ _Pessimism_ is a public good monitoring service that allows for Optimism bedrock
...
+Pessimism is currently experimental and very much in development. It means Pessimism is currently unstable, so code will change and builds can break over the coming months. If you come across problems, it would help greatly to open issues so that we can fix them as quickly as possible.
 
 ## Setup
-1. Create local config file (`config.env`)
-    * `cp config.env.template config.env`
+1. Create local config file (`config.env`) to store all necessary environmental variables. There's already an example `config.env.template` in the repo that stores default env vars.
 
-# TBD
+To use the template, run the following the command(s):
+* `cp config.env.template config.env`
```
Comment (nadir-akhtar-coinbase) [inline README.md:27]:
```suggestion
1. Create local config file (`config.env`) to store all necessary environmental variables. There's already an example `config.env.template` in the repo that stores default env vars.
```

**Review by nadir-akhtar-coinbase — COMMENTED** [review]
@epociask Minor tweaks

**Review by nadir-akhtar-coinbase — APPROVED** [review]

**Comment by cb-heimdall** [comment]
Review Error for nadir-akhtar-coinbase @ 2023-05-02 19:52:18 UTC
User cannot review their own commit

**Review by anupsv-cb — APPROVED** [review]

**Comment by cb-heimdall** [comment]
Review Error for anupsv-cb @ 2023-05-02 22:16:44 UTC
User failed mfa authentication, see [go/mfa-help](http://go/mfa-help)

**Review by anupsv-cb — APPROVED** [review]

**Pull request merged**